client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,57 @@
+import useStyles from './useStyles';
+import { Box, Button, Typography, Select, MenuItem, OutlinedInput, Grid } from '@material-ui/core';
+import { useEffect } from 'react';
+import PinIcon from './PinIcon.png';
```

Comment:
Material ui has a large [icon library](https://v4.mui.com/components/material-icons/) that should have perfect alternatives for a location pin - they are `svg`s too! 🎉 They make working with icons very easy so definitely recommend them over the png file we have 👍🏼 

NOTE: Please also remember to remove the png file afterwards 🙏🏼